### ISSUE TITLE
[Fixed in GH-6052] Missing Visual Focus on Product List Page

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4318]
Although the "**Relevance**" select menu currently has visual focus, let's flag it so we don't lose it when fixing the other elements.

The "**Show more**" button/link currently doesn't have visual focus and is not receiving focus at all since it's not set as a link. This could be fixed with an ARIA approach.

![Screen Shot 2019-08-29 at 2 20 58 PM](https://user-images.githubusercontent.com/15113219/63967526-a16b8380-ca6b-11e9-8b9d-f5653bf5a960.png)

**Comment by KateChuen** [comment]
And for the Relevance filter, can we make sure the outline isn't missing any border please? When we click on the filter, it's missing the bottom border. 
<img width="751" alt="Screen Shot 2019-09-24 at 3 32 37 PM" src="https://user-images.githubusercontent.com/45409515/65544314-f7074480-dee0-11e9-8af2-2ae1e99ba672.png">

**Comment by developpeurweb** [comment]
**For QA:**
_When we click on the filter, it's missing the bottom border._
This has been fixed to behave as an html select field, only the top should have the outline, as options are not focusable but selectable with arrow keys.

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-31 at 4 22 04 PM](https://user-images.githubusercontent.com/15113219/73575374-0d507e80-4446-11ea-8267-27c7c6fe3458.png)
![Screen Shot 2020-01-31 at 4 21 25 PM](https://user-images.githubusercontent.com/15113219/73575375-0d507e80-4446-11ea-8882-795f777c30a7.png)
![Screen Shot 2020-01-31 at 4 22 56 PM](https://user-images.githubusercontent.com/15113219/73575376-0d507e80-4446-11ea-9e50-b06bada0f93e.png)
![Screen Shot 2020-01-31 at 4 22 46 PM](https://user-images.githubusercontent.com/15113219/73575377-0d507e80-4446-11ea-9686-a2d9820fc1a8.png)
![Screen Shot 2020-01-31 at 4 22 35 PM](https://user-images.githubusercontent.com/15113219/73575378-0d507e80-4446-11ea-9d34-af2c1a1e0947.png)
![Screen Shot 2020-01-31 at 4 22 29 PM](https://user-images.githubusercontent.com/15113219/73575379-0de91500-4446-11ea-812d-444febe5150f.png)
![Screen Shot 2020-01-31 at 4 22 14 PM](https://user-images.githubusercontent.com/15113219/73575380-0de91500-4446-11ea-96ca-5c9325746331.png)

**Comment by KateChuen** [comment]
Tested on dev3.
- Chrome ✅
- Firefox ✅
- Safari ✅

But cannot actually select some elements on the page: #6568
